src/Chapter2.hs

```diff
@@ -621,7 +640,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven []           = []
+takeEven (x : _ : xs) = x : takeEven xs
+takeEven x            = x
```

Comment:
I would recommend here to pattern match on the last remaining case explicitly to make use of GHC pattern matching checker :slightly_smiling_face: Alternatively, you can combine this case with the first line and reduce the number of lines in the implementation of this function
```suggestion
takeEven [x]          = [x]
```